test/integration/features/basic.feature

```diff
@@ -147,6 +147,12 @@ Feature: Basic
      """
      exposed
      """
+
+  Scenario: User can access ruby-ex route
```

Comment:
Considering ```"/"``` as any service and ```foo``` as projectname 

If i understand this correctly, the aim is to get the HTTP response of a application deployed in the name space irrespective of service name.

> that one might want to hit something else than the root context "/".

@hferentschik i don't understand when you say hit something else. Are you referring it to any service in the same project ? bit of explanation would be really helpful